packages/lib-feature-flags/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Could you please explain why do you need this file here?

Reply:
@miadzvedski-dzmitry Because without it lint is failing :(